### PR TITLE
Feat/add illustrations

### PR DESCRIPTION
**Pull request by ccamel**
Add several illustrations to the whitepaper.

(based on the work of @ManonPoint26)

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -70,6 +70,30 @@ jobs:
...
 
+  lint-image:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Lint image files
+        run: |
+          find "$FOLDER" -regextype posix-egrep -regex '.+\.(jpg|jpeg|png|webp)' -exec sh -c '
```
Comment (ad2ien) [inline .github/workflows/lint.yml:81]: Nice! Maybe we can add `ico` and `svg` files

```suggestion
          find "$FOLDER" -regextype posix-egrep -regex '.+\.(ico|jpg|jpeg|png|svg|webp)' -exec sh -c '
```

**Review by ad2ien — APPROVED** [review]

**Comment by fredericvilcot** [comment]
@ccamel 
Great work!
I have an interrogation: why using `png` format instead of `webp`?

**Comment by ccamel** [comment]
@fredericvilcot We keep the images in `png` on this PR. @jeremylgn  will convert all the images from `png` to `webp` in another PR.

**Comment by ccamel** [comment]
@fredericvilcot if everything is good for you, we can merge this PR.

**Comment by fredericvilcot** [comment]
> @fredericvilcot We keep the images in `png` on this PR. @jeremylgn will convert all the images from `png` to `webp` in another PR.

LGTM, thx 😌

**Review by fredericvilcot — APPROVED** [review]
👌

**Pull request merged**